core/themes/basis/template.php

```diff
@@ -8,11 +8,24 @@
  * Implements hook_css_alter().
  */
 function basis_css_alter(&$css) {
-  // Remove Basis' `/css/component/menu-dropdown.css` if using a custom
-  // breakpoint.
-  if (config_get('menu.settings', 'menu_breakpoint') == 'custom') {
+  // Remove Basis' `/css/component/menu-dropdown.css` and add breakpoint files
+  // if using a custom breakpoint.
+  $config = config('menu.settings');
+  if ($config->get('menu_breakpoint') == 'custom') {
     $path = backdrop_get_path('theme', 'basis');
+    $weight = isset($css[$path . '/css/component/menu-dropdown.css']['weight']) ? $css[$path . '/css/component/menu-dropdown.css']['weight'] : 0;
```

Comment:
Should this whole conditional be hinged of if this CSS file is added to the page?
```suggestion
  if (isset($css[$path . '/css/component/menu-dropdown.css']) && $config->get('menu_breakpoint') == 'custom') {
    $dropdown_css = $css[$path . '/css/component/menu-dropdown.css'];
    $path = backdrop_get_path('theme', 'basis');
```

Then reference `$dropdown_css` in the calculations below? That would look cleaner than using `$css[$path . '/css/base.css']` as the template.

Reply:
Good idea. I was trying to think of what file would likely remain, but this is better. Don't do anything if menu-dropdown.css isn't there.